### DISCUSSION TITLE
the version of gltf-transform for nodejs10.14.0??

### DISCUSSION THREAD

**Discussion by zw-sheng** [donmccurdy/glTF-Transform#1456] [Q&A]
I use nodejs10.14.0 for my project, Dose gltf-transform support Node.js10.14.0, and if so, which version should I chose?

**Comment by donmccurdy** [comment]
Hi @zw-sheng! glTF Transform aims to support [maintained LTS releases](https://endoflife.date/nodejs) of Node.js — currently that's v18, v20, and v22. You might be able to get it to work as far back as v14, depending on what features you're using. But I suspect you'll need a node.js version newer than v10, unless you're OK building the library from source and fixing issues that may arise.

**Reply by zw-sheng** [reply]
Thank you so much, I understand.
